### PR TITLE
Show clock in TV layout

### PR DESCRIPTION
**Pull request by addamanth**
Show clock in TV layout

**Changes**
Show clock in TV layout in header 

**Screenshots**
scr1
scr2

### CODE REVIEW

**Comment by thornbill** [comment]
It would be nice to include a screenshot in the PR description for a UI change like this. :slightly_smiling_face:

**Comment by addamanth** [comment]
> It would be nice to include a screenshot in the PR description for a UI change like this. 🙂

Screenshots attached

**Review by thornbill — CHANGES_REQUESTED** [review]
Looks great! Just one minor suggestion.

File: src/scripts/libraryMenu.js [hunk 1]
```diff
@@ -189,6 +193,17 @@ import '../styles/flexstyles.scss';
         }
     }
 
+    function updateClock() {
+        if (layoutManager.tv) {
+            currentTimeText.classList.remove('hide');
+            setInterval(function() {
+                 currentTimeText.innerHTML = datetime.getDisplayTime(new Date());
```
[the remark this answers is not in the document]
Comment (addamanth) [reply src/scripts/libraryMenu.js]: Thank you for the feedback!
PR updated with the suggested change

**Comment by DrConflict** [comment]
Could the clock also be added to the screensaver by any chance? Really looking forward to having this feature added in 10.9!

**Pull request merged**